packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -611,7 +607,11 @@ class FlutterLocalFileComparator extends FlutterGoldenFileComparator with LocalC
       baseDirectory.createSync(recursive: true);
     }
 
-    goldens ??= SkiaGoldClient(baseDirectory);
+    goldens ??= SkiaGoldClient(baseDirectory,
```

Comment:
The local comparator is used in a non-ci environment, so neither of these would be correct. 🤔

Reply:
WDUT about `ContinuousIntegrationEnvironment.none` ?

Reply:
SGTM. The switch statements will want it to be accounted for, it'll just be a pass through then. :) Thanks!